suspicious_tlds.txt

```diff
@@ -1,456 +1,65 @@
-0-0-0.info
-01.ca
-04.mn
-0d.eu
-0m.la
-0rg.fr
-10.ca
-10.nl
-1s.fr
-2009.tk
-2010.tk
-22.su
-2y.net
-345.pl
-3v.eu
-4000.tk
-4f.es
-4n.jp
-51.no
-7000.tk
-88n.eu
-89.me
-8s.nl
-9000.tk
-aaa.ai
-ab.im
 accountants
 ac.th
-ad.mn
-ae.am
-ae.tn
-air.ai
-aj.ru
 am
-am.cx
-an.cl
-apg.org
-aq.pl
-at.gg
-at.hm
-at.lv
-at.mn
-at.pn
-at.tc
-at.tf
-at.vc
-at.vu
-atnet.nl
-au.mn
-au.ms
-au.pn
-au.tc
-au.us
-bd.tn
-be.ma
-be.tc
-bee.pl
-bg.tc
-bg.tf
-bi.sh
-bij.pl
-biz.br.tc
-biz.ly
-bo.rs
-box.tl
-br.ms
-br.tc
-bt.ch
-byinter.net
-bz.cm
-c0m.at
-c0m.li
-c4.fr
-ca.pn
-ca.tf
-casubi.com
-cc.ai
-cc.cc
-cd.am
-ce.ms
-ce.vg
 cf
-cg.hk
-ch.gg
-ch.lv
-ch.ma
-ch.pn
-ch.tc
-ch.tf
-ch.vc
-ch.vu
-check-it-out.nl
 cl
 click
 club
 cm
-cn.mn
-cn.ms
-cn.pn
-cn.tn
-co.at.gg
-co.at.lv
-co.at.mn
-co.at.pn
-co.at.tc
-co.at.tf
-co.at.vc
-co.be
-co.cc
 co.cl
-co.com.au
-co.nf
-co.nr
-co.nu
-co.tv
-co.uk.mn
-co.uk.pn
-co.uk.tc
-co.vu
 co.za
-com.au.mn
```

Comment:
some of these surprised me but I confirmed they aren't in the public suffix list and it's consistent with how we parse domains for the MDM

https://publicsuffix.org/list/public_suffix_list.dat